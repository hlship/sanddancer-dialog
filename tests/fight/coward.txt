(setup chase)
(now) (knock holds #courage)
take jade
n
w
(now) (#chase has completed)
z
n
z
z
z
z
z
%% That removes the coyotes, one more tick to end the scene:
z


