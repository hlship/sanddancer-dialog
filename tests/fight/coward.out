

Middle of Nowhere (in the pickup truck)
The tire tracks from the south stop abruptly here, but where the hell are you?
The desert sand and clumps of pale sagebrush are all your dimming headlights
pick out before barely reaching the concrete building to the north.

Your poor old pickup truck ticks and groans, smashed gracelessly against a tall
Saguaro; wiffs of evaporating gasoline linger in the chill air. On the dash you
can see a piece of jade. The glove compartment is closed.

Through the windshield, you see a little lizard clinging frantically to the
Saguaro.

> (setup chase)
Query succeeded: (setup chase)
> (now) (knock holds #courage)
> take jade
You take the piece of jade.

Something about the piece of jade makes you want to brood about your grandma's
stories.

> n
(first attempting to get out of the pickup truck)
You work the rusty truck door open. You get out of the pickup truck.

You walk north.

Crumbling Concrete
This building must've been some sort of utility structure for the huge
electrical tower to the north, now abandoned. Tendrils of desert sand drift
against a boarded-up door to the northeast leading inside; you could also walk
around to the east or head south back to your truck.

> w
You walk west for a long time, thoughts turning over like the sand under your
sneakers. You think about your grandma's stories. You think about Ocean and how
pissed she's gonna be that you're not home. Like you tell her not to wait up for
you but she always does anyway.

Some wind kicks up and chucks sand in your face. You blink it out and kinda
realize while you're doing it that you haven't really been paying attention to
where you've been going. Another gust whines through the sagebrush and you
realize with a kind of heavy feeling that with all the dust this wind is kicking
up you can't see a damn thing. You can't see your pickup, you can't see the
electrical tower. Not even your damn footprints. You're lost. Well, shit.

Open Desert
Clouds of dust swirl through the air, and every inch of sand looks just like
every other.

Half-lost in the shadows, you see a stand of cactus, a low bluff, and some dry
bones and that's about it.

> (now) (#chase has completed)
> z
Nah, man, better keep busy or else it's back to brooding about your grandma's
stories.

It's over. They've caught you. They surround you, circling, growling, looking
for weakness, moving in for the kill.

> n
You try to flee and instantly teeth and claws block your path; you pull back to
the center of the circle, terrified.

A coyote snaps his teeth, lunging foward, then fading back into the pack.

> z
Nah, man, better keep busy or else it's back to brooding about your grandma's
stories.

The circle of snarling coyotes tightens.

> z
Nah, man, better keep busy or else it's back to brooding about your grandma's
stories.

A coyote snaps his teeth, lunging foward, then fading back into the pack.

> z
Nah, man, better keep busy or else it's back to brooding about your grandma's
stories.

The coyotes growl, pulling closer, snapping at your heels.

> z
Nah, man, better keep busy or else it's back to brooding about your grandma's
stories.

A coyote snaps his teeth, lunging foward, then fading back into the pack.

> z
Nah, man, better keep busy or else it's back to brooding about your grandma's
stories.

The circle of snarling coyotes tightens.

> z
Nah, man, better keep busy or else it's back to brooding about your grandma's
stories.

The coyotes growl, pulling closer, snapping at your heels.

It's over. The coyotes dive in for the kill, jaws snapping, and you cower in
fear.

But then one of them flicks his head and bares his teeth, and the others pull
back. He's not bigger or stronger or anything, but there's something about him
that's different. He takes the littlest step forward and the others scatter,
mangy shadows slinking away into the desert and then you're left alone with the
leader, the alpha coyote.

And then you blink, and realize he's not a coyote after all. Just... just a guy.

He wears a couple of days of stubble, a scuffed and faded denim jacket, and,
ridiculously in the darkness, a pair of cheap sunglasses. He lights a cigarette,
then holds up a hand in annoyance to block your flashlight beam. "Turn that
thing off," he says, and either you do it or it goes off by itself, because the
next thing you know, it's darker and he's closer and the only light comes from
the red glow of the cigarette.

"That's better," he says. He takes a drag on the cigarette, then pulls another
from behind his ear and offers it to you. "Smoke?"

You could say yes or say no.

> 
